auxx/src/Plugin.hs

```diff
@@ -79,10 +79,12 @@ rawExec mHasAuxxMode AuxxOptions{..} mSendActions = \case
         printAction <- getPrintAction
         let commandProcs = createCommandProcs mHasAuxxMode printAction mSendActions
         printAction "... the auxx plugin is ready"
-        forever $ withCommand $ \line -> do
-            expr <- eitherToThrow $ Lang.parse line
-            value <- eitherToThrow =<< Lang.evaluate commandProcs expr
-            withValueText printAction value
+        forever $ withCommand $ \line -> case Lang.parse line of
+            Left parseError -> printAction (show . Lang.ppParseError $ parseError)
```

Comment:
What about updating `runWalletCmd` accordingly? Move this logic to a helper, and remove `Exception` instances for `ParseError` and `EvalError`